src/components/MerchantPayout/VoucherStatusModal/InvalidCard.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FunctionComponent } from "react";
+import { StyleSheet, View } from "react-native";
+import { Card } from "../../Layout/Card";
+import { AppText } from "../../Layout/AppText";
+import { DarkButton } from "../../Layout/Buttons/DarkButton";
+import { size, color, fontSize } from "../../../common/styles";
+import { MaterialIcons } from "@expo/vector-icons";
+
+const styles = StyleSheet.create({
+  card: {
+    width: 512,
+    maxWidth: "100%",
+    borderTopWidth: size(1),
+    backgroundColor: color("red", 10),
+    borderColor: color("red", 50)
+  },
+  iconWrapper: {
+    marginBottom: size(1.5),
+    marginTop: size(1)
+  },
+  headerText: {
+    fontFamily: "brand-bold",
+    fontSize: fontSize(3),
+    marginBottom: size(2.5)
+  }
+});
+
+interface InvalidCard {
+  title: string;
+  details?: string;
```

Comment:
Are you thinking of a case when we wont show details?

Reply:
Yep, we were unsure if there will be other details in the future but we will amend it!